src/communication/xcWebSocketSubscriber.ts

```diff
@@ -194,7 +194,7 @@ export class DefaultSubscriber implements Subscriber {
     }
 
 
-    getStateMachineUpdates(componentName: string, stateMachineName: string): any {
+    getStateMachineUpdates(componentName: string, stateMachineName: string): Observable<any> {
```

Comment:
we can improve prepareStateMachineUpdates return type too !
Do we still need any (in the observable) ? 
We can define a jsonData type for instance

Reply:
resolvse #97